scm/driver/bitbucket/content.go

```diff
@@ -20,10 +20,18 @@ func (s *contentService) Find(ctx context.Context, repo, path, ref string) (*scm
 	endpoint := fmt.Sprintf("/2.0/repositories/%s/src/%s/%s", repo, ref, path)
 	out := new(bytes.Buffer)
 	res, err := s.client.do(ctx, "GET", endpoint, nil, out)
-	return &scm.Content{
+	content := &scm.Content{
 		Path: path,
 		Data: out.Bytes(),
-	}, res, err
+	}
+	if err != nil {
+		return content, res, err
+	}
+	metaEndpoint := fmt.Sprintf("/2.0/repositories/%s/src/%s/%s?format=meta", repo, ref, path)
+	metaOut := new(metaContent)
+	metaRes, err := s.client.do(ctx, "GET", metaEndpoint, nil, metaOut)
```

Comment:
have your confirmed meta API call works for pull request and tag refs?  Also, I was wondering if a failure to fetch the meta should result in an error or not.  Is there any risk that returning the file may succeed but returning the meta may fail ?  This is me just being extra cautious because Drone relies on this function, but we would not want a meta failure from preventing a pipeline from running.